modules/logging-release-notes-5-6-17.adoc

```diff
@@ -0,0 +1,26 @@
+// module included in /logging/logging-5-6-release-notes
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-6-17_{context}"]
+= Logging 5.6.17
+This release includes link:https://access.redhat.com/errata/RHSA-2024:1507[OpenShift Logging Bug Fix 5.6.17]
```

Comment:
In your original PR, I missed that this is missing a period. I'll merge it for now, but if you do any updates to this file in the future, would you mind adding it in across versions?

Reply:
Sure thing! I have some grooming to do for Logging z-stream releases anyway. Will do it as a part of that task.